### ISSUE TITLE
Add thumb cleanup interval configuration to Imagine plugin

### ISSUE THREAD

**Issue by espaan** [zikula/core#1644]
The Imagine plugin has the option to automatically cleanup the generated thumbnails every day. This datetime should be configurable. This can be for instance twice per day or only once every week for less visited sites.

**Comment by Guite** [comment]
This is a won't fix, because the imagine plugin is deprecated and being replaced by LiipImagineBundle. http://symfony.com/doc/current/bundles/LiipImagineBundle/index.html
